flake8 errors happen during EPEL 7 build

The version of flake8 used in EPEL 7 is upset about a few things:

```
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/server/buildsys.py:252:17: E126 continuation line over-indented for hanging indent
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/server/services/releases.py:64:13: E126 continuation line over-indented for hanging indent
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/server/services/releases.py:68:13: E126 continuation line over-indented for hanging indent
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/server/views/generic.py:224:101: E501 line too long (116 > 100 characters)
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/server/consumers/masher.py:558:33: E126 continuation line over-indented for hanging indent
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/tests/server/consumers/test_masher.py:104:101: E501 line too long (112 > 100 characters)
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/tests/server/consumers/test_masher.py:399:18: E121 continuation line indentation is not a multiple of four
```

We could either find a way to appease both versions of flake8, or we could adjust the style test to detect which version of flake8 is present and only test with the newer one that we like. Doesn't matter to me.

--- a/bodhi/server/buildsys.py
+++ b/bodhi/server/buildsys.py
@@ -248,13 +248,13 @@ class DevBuildsys(Buildsystem):
 
     def getRPMHeaders(self, rpmID, headers):
         return {
-            'description':
+            'description': (
                 "The libseccomp library provides an easy to use interface to the "
                 "Linux Kernel's\nsyscall filtering mechanism, seccomp. The "
                 "libseccomp API allows an application\nto specify which "
                 "syscalls, and optionally which syscall arguments, the\n"
                 "application is allowed to execute, all of which are "
-                "enforced by the Linux\nKernel.",
+                "enforced by the Linux\nKernel."),
             'url': 'http://libseccomp.sourceforge.net',
             'changelogname': [
                 'Fedora Release Engineering <rel-eng@lists.fedoraproject.org> - 2.1.0-1',

--- a/bodhi/server/consumers/masher.py
+++ b/bodhi/server/consumers/masher.py
@@ -552,10 +552,8 @@ class MasherThread(threading.Thread):
                             from_tag = tag
                             break
                     else:
-                        reason = (
-                            'Cannot find relevant tag for %s.  '
-                            'None of %s are in %s.') % (
-                                build.nvr, tags, tag_types[status])
+                        reason = 'Cannot find relevant tag for %s.  None of %s are in %s.'
+                        reason = reason % (build.nvr, tags, tag_types[status])
                         self.eject_from_mash(update, reason)
                         break
 

--- a/bodhi/server/services/releases.py
+++ b/bodhi/server/services/releases.py
@@ -59,13 +59,11 @@ def get_release_html(request):
     if not release:
         request.errors.add('body', 'name', 'No such release')
         request.errors.status = HTTPNotFound.code
-    updates = request.db.query(Update).filter(
-        Update.release == release).order_by(
-            Update.date_submitted.desc())
+    updates = request.db.query(Update).filter(Update.release == release).order_by(
+        Update.date_submitted.desc())
 
     updates_count = request.db.query(Update.date_submitted, Update.type).filter(
-        Update.release == release).order_by(
-            Update.date_submitted.desc())
+        Update.release == release).order_by(Update.date_submitted.desc())
 
     date_commits = {}
     dates = set()

--- a/bodhi/server/views/generic.py
+++ b/bodhi/server/views/generic.py
@@ -221,7 +221,8 @@ def api_version(request):
 def notfound_view(context, request):
     """ Automatically redirects to slash-appended routes.
 
-    http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html#redirecting-to-slash-appended-rou
+    Note: The URL below spans two lines.
+    http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html
     """
     return exception_view(context, request)
 

--- a/bodhi/tests/server/consumers/test_masher.py
+++ b/bodhi/tests/server/consumers/test_masher.py
@@ -101,7 +101,9 @@ class TestMasher(unittest.TestCase):
         # The BUILD_ID environment variable is set by Jenkins and allows us to
         # detect if
         # we are running the tests in jenkins or not
-        # https://wiki.jenkins-ci.org/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-below
+        # Note: The URL below spans two lines.
+        # https://wiki.jenkins-ci.org/display/JENKINS/Building+a+software+project
+        # #Buildingasoftwareproject-below
         if os.environ.get('BUILD_ID'):
             faitout = 'http://209.132.184.152/faitout/'
             try:
@@ -396,7 +398,7 @@ References:
              'https://bugzilla.redhat.com/show_bug.cgi?id=12345\n  [ 2 ] CVE-1985-0110\n        '
              'http://www.cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-1985-0110\n--------------------'
              '------------------------------------------------------------\n\n') % (
-                 config.get('fedora_test_announce_list'), time.strftime('%Y'))), repr(body)
+                config.get('fedora_test_announce_list'), time.strftime('%Y'))), repr(body)
 
     def test_sanity_check(self):
         t = MasherThread(u'F17', u'testing', [u'bodhi-2.0-1.fc17'],

